src/GADM.jl

```diff
@@ -122,4 +123,25 @@ function get(country, subregions...)
     throw(ArgumentError("feature not found"))
 end
 
+"""
+    coordinates(country, levels...)  
+
+Returns a deep array of coordinates for the requested region.  
+Input: ISO 3166 Alpha 3 Country Code, and further full official names of subdivisions  
+
+## Example:
+  
+```julia
+julia> coordinates("VAT") # Returns a deep array of Vatican city
```

Comment:
Shouldn't this example be `GADM.coordinates("VAT")`?

Reply:
Please fix this, and possibly the examples in GADM.get, and feel free to merge afterwards.